docs/iris/src/whatsnew/2.3.rst

```diff
@@ -0,0 +1,193 @@
+What's New in Iris 2.3.0
+**********************
+
+:Release: 2.3.0
+:Date: 2019-10-04
+
+This document explains the new/changed features of Iris in version 2.3.0
+(:doc:`View all changes <index>`.)
+
+Iris 2.3.0 Features
+=================
+.. _showcase:
+
+.. admonition:: Increased Support for CF 1.7
+
+  Several changes have contributed to our increased support for CF 1.7:
+
+  * climatological coordinate support
+  * standard name modifier support
+  * name tokens enforcement in cell methods
+  * coordinate var_name, CF_CONVENTIONS_VERSION advanced to 1.7
+
+  You can read more about each of these below.
+
+.. admonition:: Climatological Coordinate Support
+
+  Iris can now load, store and save `NetCDF climatological coordinates
+  <http://cfconventions.org/Data/cf-conventions/cf-conventions-1
+  .7/cf-conventions.html#climatological-statistics>`_. Any cube time
+  coordinate can be marked as a climatological time axis using the boolean
+  property: ``climatological``. The climatological bounds are stored in the
+  coordinate's ``bounds`` property.
+
+  When an Iris climatological coordinate is saved in NetCDF, the NetCDF time
+  axis will be given a 'climatology' attribute, and the contents of the
+  coordinate's ``bounds`` property are written to a NetCDF boundary variable
+  called 'climatology_bounds'.  These are in place of the more usual NetCDF
+  'bounds' attribute and accompanying 'bnds' boundary variable. See below
+  for an example:
+
+    .. code-block::
```

Comment:
This errors when building:
`2.3.rst:41: WARNING: Error in "code-block" directive:`
And the resulting built page misses the code block completely. 

```suggestion
    .. code-block:: none
```

Reply:
Interesting that this worked on my local build without "none". Anyway I've made this change and the new formatting still looks good 👍

Reply:
This may be because I'm using an old version of sphinx. I think more recent versions default to python highlighting

Reply:
It is now allowed to omit the argument in more recent versions of sphinx
https://github.com/sphinx-doc/sphinx/pull/6015
I just need to update my environment!